site/src/pages/UsersPage/UsersFilter.tsx

```diff
@@ -58,6 +59,8 @@ export const UsersFilter = ({
     <Filter
       presets={PRESET_FILTERS}
       learnMoreLink={docs("/admin/users#user-filtering")}
+      learnMoreLabel2="User status"
+      learnMoreLink2={docs("/admin/users#user-status")}
```

Comment:
I don't think this link makes sense here because it is not related to the filter but we can think on that later tho

Reply:
Don't worry about that. I will re-think a few things for the user action menu.